custom-login/package.json

```diff
@@ -8,29 +8,30 @@
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
     "test": "npm run lint && npm run test:custom-login --prefix ../",
-    "pretest": "npm run pretest --prefix ../"
+    "pretest": "npm run pretest --prefix ../",
+    "postinstall": "npm run patch-semantic-ui-css --prefix ../"
   },
   "dependencies": {
     "@okta/okta-auth-js": "^6.2.0",
     "@okta/okta-signin-widget": "^6.4.0",
     "@okta/okta-vue": "^5.1.0",
-    "axios": "^0.21.1",
+    "axios": "^0.27.2",
     "core-js": "^3.15.1",
-    "semantic-ui-css": "^2.4.1",
+    "fomantic-ui-css": "^2.8.8",
     "text-encoding": "^0.7.0",
     "vue": "^3.1.2",
     "vue-final-modal": "^3.4.3",
     "vue-router": "^4.0.10"
   },
   "devDependencies": {
-    "@vue/cli-plugin-babel": "~4.5.0",
-    "@vue/cli-plugin-eslint": "~4.5.0",
-    "@vue/cli-plugin-router": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
+    "@vue/cli-plugin-babel": "^5.0.8",
```

Comment:
nit: keep `~` notation rather than `^`, just so we don't pull in unexpected changes